module/solutions/helper.js

```diff
@@ -963,7 +976,13 @@ module.exports = class SolutionsHelper {
 					filterQuery['scope.roles'] = {
 						$in: [CONSTANTS.common.ALL_ROLES, ...data.role.split(',')],
 					}
-					filterQuery['scope.entities'] = { $in: entities }
+
+					filterQuery.$or = []
+					Object.keys(_.omit(data, ['filter', 'role', 'factors'])).forEach((key) => {
```

Comment:
@Prajwal17Tunerlabs type is also  missing from here. check if we have to add it

Reply:
have added type keyword.